src/DevBetterWeb.Web/Services/WebVTTParsingService.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace DevBetterWeb.Web.Services;
```

Comment:
I think this service needs to be in the Core project.

Reply:
Base on @ardalis' [comments](https://github.com/DevBetterCom/DevBetterWeb/pull/926#issuecomment-1246997543) I will keep the service in the Web project.